### PR TITLE
[mieb] run tasks fix

### PR DESCRIPTION
**Pull request by gowitheflow-1998**
1. fix big task type so that can grab mieb tasks with mteb.get_tasks
2. fix dataloader issues for vista and e5-v.
3. add script to run mieb tasks.

- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`.

### CODE REVIEW

File: scripts/run_mieb.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+import mteb
+
+for model_name in [
```
Comment (isaac-chung) [inline scripts/run_mieb.py:5]: What about
- datacomp clip
- moco models

File: scripts/run_mieb.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            "ImageClustering",
+            "ImageClassification",
+            "ImageMultilabelClassification",
+            "ImageTextPairClassification",
+            "VisualSTS",
+            "ZeroShotClassification",
+        ]
+    )
+    evaluation = mteb.MTEB(tasks=tasks)
+    results = evaluation.run(model, output_folder="results-mieb-final")
```
Comment (isaac-chung) [inline scripts/run_mieb.py:40]: What should we do with the existing results in `results-mieb`?

**Review by isaac-chung — COMMENTED** [review]
Looking good! A few small questions.

File: scripts/run_mieb.py (hunk 1 above)
Comment (isaac-chung) [reply scripts/run_mieb.py:5]:
```suggestion
for model_name in [
    "laion/CLIP-ViT-L-14-DataComp.XL-s13B-b90K",
    "laion/CLIP-ViT-B-32-DataComp.XL-s13B-b90K",
    "laion/CLIP-ViT-B-16-DataComp.XL-s13B-b90K",
    "nyu-visionx/moco-v3-vit-b",
    "nyu-visionx/moco-v3-vit-l",
```

Comment (gowitheflow-1998) [reply scripts/run_mieb.py:5]: have added!

**Pull request merged**